docker-storage-setup.conf

```diff
@@ -84,17 +84,20 @@ DEVICE_WAIT_TIMEOUT=60
 # as this means if there was any leftover data on disk, it will be lost.
 WIPE_SIGNATURES=false
 
-# By default no new volume and filesystem will be setup for docker
-# root dir. Docker creates /var/lib/docker/ on top of underlying filesystem
-# for storing images and containers.
+# By default no new volume and filesystem will be setup
+# for container root directory.
 #
-# To carve out a separate logical volume for storing docker
-# images/containers/volumes data, set DOCKER_ROOT_VOLUME=yes
-# NOTE: devicemapper stores images/containers on thin pool.
+# To carve out a separate logical volume for storing
+# images/containers/volumes data, set ROOT_VOLUME_MOUNT_DIR="PATH"
+# where "PATH" will be fully qualified path of the directory where
+# you want to store your data. e.g. ROOT_VOLUME_MOUNT_DIR=
+# "/var/lib/containers" would create a logical volume and mount it
+# on "/var/lib/containers". NOTE: devicemapper stores images/containers
+# on thin pool.
 #
```

Comment:
How about following.

`Create a logical volume and mount it on specified directory path. By default no new logical volume will be created.`
`Example. Specifying ROOT_VOLUME_MOUNT_DIR=/var/lib/container will carve out a logical volume, create filesystem on it and mount it on /var/lib/containers` 

`